guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

```diff
@@ -3,6 +3,8 @@
 
 = Attaching the {Project} Infrastructure Subscription
 
+NOTE: Skip this section if the Simple Content Access (SCA) is already in use.
```

Comment:
```suggestion
NOTE: Skip this section if you are already using the Simple Content Access (SCA).
```
Active voice :)

Reply:
Or even better (as [in 6.11](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_capsule_server/installing-capsule-server#attaching-infrastructure-subscription_capsule))
```suggestion
NOTE: Skip this section if you enabled the Simple Content Access (SCA) in the Red{nbsp}Hat Customer Portal.
```